lib/msal-node/test/test_kit/StringConstants.ts

```diff
@@ -348,6 +350,22 @@ export const AUTHENTICATION_RESULT_DEFAULT_SCOPES = {
     },
 };
 
+export const MANAGED_IDENTITY_RESOURCE =
+    "URI_of_the_managed_identity_assigned_to_the_azure_resource";
+
+export const MANAGED_IDENTITY_CLIENT_ID =
+    "unique_identifier_generated_by_Azure_AD_for_the_azure_resource";
+
+export const DEFAULT_MANAGED_IDENTITY_AUTHENTICATION_RESULT: ManagedIdentityResult =
+    {
+        accessToken: TEST_TOKENS.ACCESS_TOKEN,
+        expiresOn: TEST_TOKEN_LIFETIMES.DEFAULT_EXPIRES_IN,
+        resource: MANAGED_IDENTITY_RESOURCE,
```

Comment:
Is "resource" new? I thought AuthResult only has scopes.

Reply:
not returning resource anymore since the developer already knows it